research/flamby/README.md

```diff
@@ -33,8 +17,56 @@ cd <fl4health_repository>
 pip install --upgrade pip poetry
 poetry install --with "dev, dev-local, test, codestyle"
 cd <FLamby_repository>
-pip install -e ".[all_extra]"
+pip install -e ".[cam16, heart, isic2019, ixi, lidc, tcga]"
+```
+__NOTE__: We avoid installing Fed-KITS2019, as it requires a fairly old version on nnUnet, which we no longer support in our library.
+
+In addition, you'll have to edit the code for `FedIXITiny` in `flamby/datasets/fed_ixi/datasets.py` replacing the following
```

Comment:
The work around below is required in order to run any of the experiments, since our dataset processing files handle all of the dataset preprocessing for the datasets of interest. It's pretty straightforward to do since we install the library as "editable" (i.e. the -e) option.

Reply:
FLamby corrected a few things with MonAI compatibility but apparently left this one behind.